applications/matter_weather_station/README.rst

```diff
@@ -7,7 +7,7 @@ Thingy:53: Matter weather station
    :local:
    :depth: 2
 
-This Matter weather station application demonstrates the usage of the :ref:`Matter <ug_matter>` application layer to build a weather station device.
```

Comment:
This file is outdated and does not include latest edits. Please update PR with the latest .rst file.